terraform/environments/apex/aws_waf_ipset.txt

```diff
@@ -161,4 +161,4 @@
 51.149.249.32/29
 194.33.248.0/29
 20.49.214.199/32
-20.49.214.228/32
\ No newline at end of file
+20.49.214.228/32
```

Comment:
@ffteva-moj you you revert all changes to the apex file please so the PR is clean? I'm not sure if its whitespace or a newline but something is making git think there is a change still